Makefile

```diff
@@ -11,7 +11,6 @@ deps:
 	which godep || go get github.com/tools/godep
 
 build: clean deps
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go install ./...
```

Comment:
I think we still want this. now there's nothing verifying that everything in the repo builds.